src/js/drop-down.js

```diff
@@ -54,18 +60,26 @@ class DropDown {
 		relatedContent.forEach(item => shiftItems ? navList.appendChild(item) : relatedContentList.appendChild(item));
 	}
 
-	_swapClasses (existingClass, newClass, tabbing) {
+	_swapClasses (existingClass, newClass, expand) {
```

Comment:
This method name `_swapClasses` is a little confusing as its role is larger, I think we discussed something like `toggleDropdown`?  It would only need the expand argument then too.

Reply:
unfortunately that doesn't work, because the class that needs the time out is different if the nav is being dropped or raised. I've changed the name of the function, but it can't be slimmed down to a `this.nav.classList.toggle`.

The most we could do is repeat all of the code in a conditional?